locales/sr_RS/LC_MESSAGES/duckduckgo.po

```diff
@@ -608,37 +614,44 @@ msgid ""
 "Browse as usual while we add privacy protection. We bundled our search "
 "engine, tracker blocker, and encryption enforcer into one %s%s Extension%s."
 msgstr ""
+"Претражујте као и обично док додамо заштиту приватности. Наш претраживач, "
+"блокатор трагача и извршилац шифровања спојили смо у један %s%s додатак%s."
 
 msgid ""
 "Browse as usual, and we'll take care of the rest. We bundled our search "
 "engine, tracker blocker, and encryption enforcer into one %s%s Extension%s."
 msgstr ""
+"Претражујте као и обично, а ми ћемо се побринути за остало. Наш претраживач, "
+"блокатор трагача и извршилац шифровања спојили смо у један %s%s додатак%s."
 
 msgid ""
-"Browse as usual, and we’ll take care of the rest. Get bundled private "
-"search, tracker blocking, and site encryption, all in one download, for "
-"%smajor browsers%s."
+"Browse as usual, and we’ll take care of the rest. Get bundled private search,"
+" tracker blocking, and site encryption, all in one download, for %smajor "
+"browsers%s."
 msgstr ""
+"Претражујте као и обично, а ми ћемо се побринути за остало. Приватну "
+"претрагу, блокатор трагача и шифровање сајтова спојите у један %s%s "
+"додатак%s."
```

Comment:
Do you mind taking a look at this again? Looks like you've used the same translation as the string above, but this is slightly different. It says `all in one download, for %smajor browsers%s` instead of `into one %s%s Extension%s`. Also note the different placeholders (those `%s`). This string only has 2, not 3. Thanks!